.gitignore

```diff
@@ -3,4 +3,6 @@ __pycache__/
 *.pyc
 .DS_Store
 run*.sh
-.vscode
\ No newline at end of file
+.vscode
+export_recs_per_day.py
```

Comment:
These seem to very specific to your machine, can we leave the `.gitignore` as is?

Reply:
You're right. This should not be part of merge request and .gitignore should stay unchanged.

Reply:
Commit rolled back